### DISCUSSION TITLE
想问问大佬们是怎么解锁steam成就系统的，有什么关于steam成就bin的共享网站吗

### DISCUSSION THREAD

**Discussion by shaoqi-li** [pjy612/SteamManifestCache#430] [Question / Answer] [answered]
我只知道解锁steam成就可以用别人导出的bin文件来解锁，记得之前大佬做的SteamManifestCache自带解锁steam成就系统功能，想问问大佬们是怎么获取这个无成就的bin文件的，有什么成就bin库或者网站有共享这个吗

**Comment by ZEERDEER** [comment]
* 用账号获取

**Comment by pjy612** [comment]
* 没有
* 用账号获取

**Reply by shaoqi-li** [reply]
这样啊，那大佬们能共享一下成就文件吗，也可以顺便在讨论区里开一个成就bin分享贴🤔😘

**Reply by pjy612** [reply]
成就不如清单密钥有用，算是锦上添花的类型。目前没有专门收集的打算

**Comment by CHESIRE721** [comment, marked as answer]
有一种方法可以获取 UserGameStatsSchema_GameID.bin 文件（其中包含所有成就数据），但问题在于生成 UserGameStats_UserID_GameID.bin 文件。部分我使用另一个没有任何记录的文件生成了这个文件，只是编辑了 GameID。问题在于有些游戏不会解锁成就，比如《Control Ultimate Edition》（870780），所以我正在寻找能够共享这些数据以验证该文件是否有所不同的人。在其他一些游戏中，这种方法是有效的。

现在，获取成就架构的方法而无需拥有游戏所有权的账户是使用 Goldberg emu 源代码中的脚本。该脚本允许你导出 UserGameStatsSchema_GameID.bin 文件（你必须手动将此文件移动到 appcache\stats 文件夹，因为 steamtools 的成就导出工具不会导入它），然后你必须人工生成 UserGameStats_UserID_GameID.bin 文件，同样也必须手动将其移动到上述文件夹。这样一来，Steam 将显示成就列表，你就可以解锁它们。但正如我所提到的，这是一个试错的过程，因为有些游戏在通知成就方面可能会花费一些时间，或者根本不会解锁它们。
我可以生成这些成就文件，几乎是无限的，只要有某个账号购买了它。
（如果我的消息不太清楚，请原谅，我正在使用AI来翻译这条消息，以便能够提供一些帮助）

**Reply by xiaohuaihai** [reply]
有没有工具可以导入自己的游玩数据然后返回steam成就解锁数据并用于手动成就的解锁。我正在使用的是successstory这个插件的添加手动成就功能。
